docs/design/job_architecture.md

```diff
@@ -7,47 +7,65 @@ Jupyter provides a "Comm" object that allows for custom messaging between the fr
 The Narrative Interface uses one of these channels to manage job information. These are funneled through an interface on the frontend side and a matching one in the kernel.
 
 ## Frontend Comm Channel
-On the frontend, there's a `jobCommChannel.js` module that uses the MonoBus system to communicate. Frontend modules use the bus system to send one of the following messages over the main channel, which then get transformed into a message that gets passed through a kernel comm object. Most of these take one or more inputs. These are listed below the command, where applicable.
+On the frontend, there's a `jobCommChannel.js` module that uses the MonoBus system to communicate. Frontend modules use the bus system to send one of the following messages over the main channel, which then get transformed into a message that gets passed through a kernel comm object.

```

Comment:
I will add some more to this documentation once I have fully implemented the frontend changes for DATAUP-497; this just brings it up to date with the current situation.